README.md

```diff
@@ -1,10 +1,10 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_task-move-spider-DOM/)
+    - [DEMO LINK](https://SerhiiSemennikov.github.io/js_task-move-spider-DOM/)
```

Comment:
Add demo link to the PR description 
<img width="949" alt="image" src="https://github.com/user-attachments/assets/d36770cb-4c06-465b-8443-a92a0283c6a3">